locales/README.md

```diff
@@ -0,0 +1,101 @@
+# Locales for the Integreat frontend
+
+## Usage
```

Comment:
Imo it is not really clear, what has to be done only for the first setup of a repo and what has to be done every time you check out a repo. Maybe you could split this. Also imo nearly everything from `Pushing changes` should be part of `Pulling changes`. The only relevant part there should be `When pushing changes you are required to do the changes directly in the `integreat-locales` repository. After that you can pull the changes.`

Reply:
I removed some redundancy at the end of pushing: https://github.com/Integreat/integreat-locales/blob/7ae42356ab25b9a5248bb9e978fa38d9fa258021/README.md#pushing-changes

I want to go over the whole process of pushing changes to the locales. This also includes getting the changes back. Also the pull command is different in both sections as I'm pulling from different branches.

Reply:
Why don't you use just a generic branch name, e.g. `<Branch>`. I still think this Readme is to complicated and contains unnecessary duplicates. I'll open a PR with suggestions.

Reply:
PR opened: https://github.com/Integreat/integreat-locales/pull/8

Reply:
Thanks I added your recommendations